.github/workflows/build-devcontainer.yml

```diff
@@ -20,4 +20,4 @@ jobs:
             -v ./.:/workspaces/plutus-tx-template \
             -w /workspaces/plutus-tx-template \
             -i ghcr.io/input-output-hk/devx-devcontainer:x86_64-linux.ghc96-iog \
-            bash -ic "cabal update && cabal run plutus-tx-template && test -e validator.uplc"
+            bash -ic "cabal update && cabal build all" 
```

Comment:
We can no longer run the programs in CI as they require changes to the haskell sources. 
This could be improved by allowing to pass e.g. `apEndTime` and `apSeller` as command line args.